packages/expo-dev-launcher/Assets.xcassets/extensions-icon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
Are we currently removing all assets in the release builds?

Reply:
I'll need to check that.

Reply:
This can be removed